pkg/xds/sync/ingress_proxy_builder.go

```diff
@@ -88,9 +94,17 @@ func (p *IngressProxyBuilder) buildZoneIngressProxy(ctx context.Context) (*xds.Z
 		}
 	}
 
+	gateways := &core_mesh.MeshGatewayResourceList{}
```

Comment:
can we just pass the `allMeshGateways` to this function?

Reply:
Will look at ingress/multizone stuff in next pr